go/common/sgx/ias/quote_test.go

```diff
@@ -10,6 +10,10 @@ import (
 )
 
 func TestQuote(t *testing.T) {
+	// TODO: Generate another testdata AVR with debug bit disabled.
```

Comment:
Can we get one? cc @pro-wh

Reply:
Does this mean AVR needs to be signed with production key? (i.e. can you turn off debug bit and sign it with testing key?)

Reply:
We cannot obtain AVR signed with the production key at the moment. Also confirmed that you cannot generate AVR with debug bit disabled without the production key. I am resolving this for now.